### PR TITLE
RHDEVDOCS-4113 - Added steps for enabling notifications

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.10, 4.11

JIRA issue:
https://issues.redhat.com/browse/RHDEVDOCS-4113

Preview pages: No preview generated due to Netlify issues.

SME+QE review: @iam-veeramalla 
Peer-review: @rolfedh

### CODE REVIEW

File: modules/gitops-argo-cd-notification.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[source,yaml]
+----
+  apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+spec:
+  notifications:
+    enabled: True
```
Comment (iam-veeramalla) [inline modules/gitops-argo-cd-notification.adoc:20]: Hi Debargo, it should be `enabled: true`

sample

```
``` yaml
apiVersion: argoproj.io/v1alpha1
kind: ArgoCD
metadata:
  name: example-argocd
spec:
  notifications:
    enabled: true
```
```

**Review by iam-veeramalla — CHANGES_REQUESTED** [review]

**Review by iam-veeramalla — APPROVED** [review]
LGTM

**Review by rolfedh — COMMENTED** [review]
Hi @DebarghoGhosh. Nice work. I've contributed a few suggestions.

**Comment by rolfedh** [comment]
Please squash to 1 commit.

**Comment by rolfedh** [comment]
Please get SME/QE review.

**Comment by DebarghoGhosh** [comment]
> Please get SME/QE review.

already got LGTM from SME

**Comment by DebarghoGhosh** [comment]
I have followed the [OpenShift Peer Review Checklist > (for writers)](https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.bp4j7064auhh)

**Review by rolfedh — APPROVED** [review]
Pre-approved to expedite merging.

**Comment by jaideepr97** [comment]
Hi @DebarghoGhosh 
The argo-cd notifications project was merged into core Argo CD a while ago
As a result, the documentation was also merged into core Argo CD documentation

**Pull request merged**